extensions/wikia/AdEngine/js/AdEngine2.js

```diff
@@ -25,6 +26,20 @@ define('ext.wikia.adEngine.adEngine', [
 		return func;
 	}
 
+	function createAdProviderContainer(providerName, slotName) {
```

Comment:
This method could clear the contents the of the slot if it exists already, so the provider gets the ready-to-go div to work on.